source/ember-inspector/data.md

```diff
@@ -1,40 +1,33 @@
-If you are using a data library that supports the Ember Inspector such
-as Ember Data, you can inspect your models by clicking on the `Data` menu.
+If you are using Ember Data, you can inspect your models by clicking on the `Data` menu.
```

Comment:
I'd like someone else's thought on this simplification. Does ember-model support Ember Inspector for example?

cc @michaelrkn